Neos.Neos/Documentation/CreatingASite/NodeTypes/DisableNodeTypes.rst

```diff
@@ -0,0 +1,35 @@
+.. _disable-nodetypes:
+
+Disable NodeTypes
+===================
+
+To hide an existing NodeType (i.e. one that comes with Neos already) you have 2 options.
```

Comment:
"i.e." doesn't mean for example but more like "that means".
"e.g." or "for example" would be more correct I think